data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+				decline
+	
+mission "Hai Leaks Response 2B"
+	landing
+	source
+		government "Republic" "Syndicate" "Free Worlds"
+		not planet
+			Deep
+			Foundry
+	to offer
+		has "Hai Leaks Response 2: active"
+	on offer
+		conversation
+			`While refuelling on <origin> you see that the media circus is continuing uabated. Apparently pirates have started making renewed attacks on trade lanes in the north, utilising some of the strange weapons observed in the recorded clip of Hai ships fighting each other. Goodness knows how they got their hands on them, but it can't be good news.`
+				decline
+
+mission "Hai Leaks Response 2C"
+	landing
+	invisible
+	source Foundry
+	to offer
+		has "Hai Leaks Response 2: active"
+	on offer
+		log "People" "Terry Adrianopolous" `Protege to the Syndicate CEO, Terry Adrianopolous has been nominated by her mentor Alastair Korban to represent the Syndicate's interests in the crisis of the Hai's secrecy. She is a very capable woman who appears well equipped for climbing the corporate ladder, including an ability to act with apparently genuine friendliness to people whom she may consider her opponents.`
+		conversation
+			`Approaching Foundry you are hailed on a private encrypted channel and directed to a private hangar where you are greeted personally by CEO Alastair Korban.`
+			`	He smiles warmly, "Apologies Captain <last>, do excuse the cloak and dagger, but in the current media climate you and I are both too well known to be seen meeting without causing the rumour mill to grind out a new story."`
+			`	"Now, allow me to introduce you to my protege Terry Adrianopolous, she will be accompanying you as the Syndicate representative."`
+			`	The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She's pleasant and approachable, and very articulate, but from your brief glimpse beforehand you have no doubt she's cut from the sort of cloth that makes her future CEO material. You're surprised that a woman would have made it this far in the Syndicate, but with your observation you resolve not to get on her bad side if at all possible.`
+			`	"Oh by the way," says Alastair, pausing briefly as he goes to depart, "The Navy and my security forces are working jointly to get some of our message relays up in the Far North to link through Ultima Thule. The pirates are wont to destroy them whenever they find them but we should be able to rely on at least basic communication to get through if you need it. Rather than flying back and forth for literally every question. That's all."`
+			branch delayedretire
+				not "free worlds reconciliation"
+			`	He leans in close to you and says quietly, "I know I said I was retiring, Terry convinced me to clean up my mess first. I am atoning for my failures, do look after her, she's a very loyal colleague."`
```

Comment:
He's within handshaking distance, and leaning in as he goes to depart? If you have a suggestion to make that clearer by all means.

Reply:
I've updated my suggestion for line 636 to try and accommodate this: instead of `goes to depart`, `turns to leave`. Also, it's never noted that he shakes your hand, which could also lead to the confusion I had.